fabric-fluid-v1/src/main/java/net/fabricmc/fabric/api/fluids/v1/math/Drops.java

```diff
@@ -0,0 +1,73 @@
+package net.fabricmc.fabric.api.fluids.v1.math;
+
+import net.minecraft.nbt.CompoundTag;
+
+/**
+ * Drops are the standard unit of fluid for the fluid api.
+ * 1/{@link #BUCKET} is the smallest amount of fluid that can be described with the Drops system.
+ * Stored in a long, at one bucket per tick, would take around 11.6 million years to overflow!
+ */
+public class Drops {
```

Comment:
quite confused on whether we are using fractions or just nunbers

Reply:
Fractions are a bad way of doing it

Reply:
fractions is still better than then number based system imo